April/30/jeff_offutt/info.yaml

```diff
@@ -0,0 +1,16 @@
+title: Jeff Offutt
+description: |
+    On April 30, 1961 was born Jeff Offutt.
+    He is an American professor of Software Engineering at George Mason University.
+    Offutt was editor-in-chief of Wiley’s journal of Software Testing, Verification and Reliability from 2007 through 2019, 
```

Comment:
Replace back-tick character "`" with a single apostrophe `'` b/c that will probably choke the existing automation script.